### PR TITLE
Don't import VRDL trees

### PR DESCRIPTION
**Pull request by ttung**
### Description
They don't exist!

Depends on #370 

#### Issue
[ch136615](https://app.clubhouse.io/genepi/story/136615/vrdl-trees-do-not-port)

### Test plan
removed all but vrdl and tulare and ran import.

### CODE REVIEW

File: scripts/migrate-all.sh [hunk 1]
```diff
@@ -38,21 +38,23 @@ done
...
         echo "Importing external samples for $county..."
         external_project_id=$(echo "$COUNTY_INFO" | jq -r ".$county".external_project_id)
         aspen-cli db --local import-covidhub-project --rr-project-id "$external_project_id" --covidhub-db-secret cliahub/cliahub_rds_read_prod  --covidhub-aws-profile biohub --aspen-group-id "$aspen_group_id"
     fi
 done
 
 for county in "${COUNTIES[@]}"; do
-    aspen_group_id=$(echo "$COUNTY_INFO" | jq -r ".$county".aspen_group_id)
-    aspen-cli db --local import-covidhub-trees --covidhub-aws-profile biohub --s3-src-prefix s3://covidtracker-datasets/cdph/"$county" --s3-key-prefix /imported/phylo_trees/"$county" --aspen-group-id "$aspen_group_id"
+    if [ "$(echo "$COUNTY_INFO" | jq ".$county.skip_tree_import")" != "true" ]; then
```
Comment (shanaxel42) [inline scripts/migrate-all.sh:56]: wait where does `$county.skip_tree_import` come from?

Comment (phoenixAja) [reply scripts/migrate-all.sh:56]: i was wondering the same thing (also about `county.aspen_group_id`), some comments would be great in this script for those of us who arn't shell experts

Comment (ttung) [reply scripts/migrate-all.sh:56]: > wait where does $county.skip_tree_import come from?

It's set for vrdl.

Comment (ttung) [reply scripts/migrate-all.sh:56]: oh wait, i missed that in the checkin. #fml

Comment (ttung) [reply scripts/migrate-all.sh:56]: > i was wondering the same thing (also about `county.aspen_group_id`), some comments would be great in this script for those of us who arn't shell experts 

Added docs in #379

**Review by shanaxel42 — APPROVED** [review]
nice!

**Pull request merged**